plotly/offline/offline.py

```diff
@@ -236,13 +247,39 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
 
     display(HTML(plot_html))
 
+    if image:
+        warnings.warn('For higher resolution images and more export '
+                      'options, consider making requests to the '
+                      'Plotly image servers. Type help(\'py.image\') '
+                      'for more details.', UserWarning)
+        # Write script to download image of the plot
+        script = ('<script>'
+                  'function downloadimage(format, height, width,'
+                  ' filename) {{'
+                  'var p = document.getElementById(\'{plot_id}\');'
+                  'Plotly.downloadImage(p, {{format: format, height: height, '
+                  'width: width, filename: filename}});'
+                  '}};'
+                  'if(document.readyState == \'complete\') {{'
+                  'downloadimage(\'{format}\', {height}, {width}, '
+                  '\'{filename}\');}}'
+                  '</script>'
+                  ).format(format=format, width=_width, height=_height,
+                           filename=filename, plot_id=plotdivid)
+        # allow time for the plot to draw
+        import time
+        time.sleep(1)
+        # inject code to download an image of the plot
+        display(HTML(script))
+
 
 def plot(figure_or_data,
          show_link=True, link_text='Export to plot.ly',
          validate=True, output_type='file',
          include_plotlyjs=True,
-         filename='temp-plot.html',
-         auto_open=True):
+         filename='temp-plot.html', auto_open=True,
+         image=False, image_filename='plot_image', format='png',
+         _width=800, _height=600):
```

Comment:
Ah sorry, I'm just abusing notation. I needed a different variable name for height and width because `height` and `width` are returned from the call to `_plot_html()` later on, and would overwrite the values set as parameters in the original plot call.